images/linux/scripts/helpers/install.sh

```diff
@@ -163,7 +163,7 @@ get_hash_from_remote_file() {
         exit 1
     fi
 
-    matching_line=$(curl -fsSL "$url" | sed 's/  / /g' | tr -d '`')
+    matching_line=$(curl -fsSL "$url" | sed 's/ \{1,\}/ /g' | tr -d '`')
```

Comment:
I would use asterisk for this:
```suggestion
    matching_line=$(curl -fsSL "$url" | sed 's/  */ /g' | tr -d '`')
```